### PR TITLE
Add organisation for each current Kedro maintainer

### PR DESCRIPTION
**Pull request by deepyaman**
> **NOTE:** Kedro datasets are moving from `kedro.extras.datasets` to a separate `kedro-datasets` package in
> [`kedro-plugins` repository](https://github.com/kedro-org/kedro-plugins). Any changes to the dataset implementations
> should be done by opening a pull request in that repository.
## Description

Second attempt at #3039

## Development notes

Modelled after https://spark.apache.org/committers.html

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [x] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [ ] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

File: docs/source/contribution/technical_steering_committee.md
Comment (deepyaman) [inline docs/source/contribution/technical_steering_committee.md:70]: @marrrcin not sure if you prefer "GetInData | Part of Xebia" or something.

File: docs/source/contribution/technical_steering_committee.md
Comment (deepyaman) [inline docs/source/contribution/technical_steering_committee.md:68]: @astrojuanlu @stichbury not sure if you prefer "QuantumBlack Labs", "QuantumBlack, a McKinsey company", or something else (for everybody).

**Review by stichbury — CHANGES_REQUESTED** [review]
I don't see Yolan here.

File: docs/source/contribution/technical_steering_committee.md
Comment (stichbury) [reply docs/source/contribution/technical_steering_committee.md:69]: QuantumBlack, AI by McKinsey with a link to https://www.mckinsey.com/capabilities/quantumblack pls.

**Comment by astrojuanlu** [comment]
Technically speaking #3061 isn't merged, right? We can hold this one until the other lands

**Comment by stichbury** [comment]
Yes, but you'll still need to add the details since otherwise how's it going to merge?

**Comment by deepyaman** [comment]
> I don't see Yolan here.

Yeah, I didn't do it because it's a separate PR. It's easy enough to update that PR or this one once one or the other merges.

File: docs/source/contribution/technical_steering_committee.md
Comment (deepyaman) [reply docs/source/contribution/technical_steering_committee.md:69]: Will do! For consistency, I will do GetInData | Part of Xebia and link that (as well as linking Claypot AI and Societe Generale with the correct accents).

File: docs/source/contribution/technical_steering_committee.md
Comment (marrrcin) [reply docs/source/contribution/technical_steering_committee.md:71]: Yes, `GetInData | Part of Xebia` is the official name we're using, thanks for noticing!

**Comment by astrojuanlu** [comment]
This can now proceed @deepyaman, there's a merge conflict

File: docs/source/contribution/technical_steering_committee.md
Comment (deepyaman) [inline docs/source/contribution/technical_steering_committee.md:71]:
```suggestion
| [Marcin Zabłocki](https://github.com/marrrcin)           | [GetInData \| Part of Xebia](https://getindata.com)                                      |
```

**Review by astrojuanlu — APPROVED** [review]
https://kedro--3067.org.readthedocs.build/en/3067/contribution/technical_steering_committee.html#current-maintainers

image

Please @marrrcin @Galileo-Galilei give your approval

**Pull request merged**
